docusaurus/README.md

```diff
@@ -0,0 +1,41 @@
+# Website
```

Comment:
Is this something you added or was it a default?

I'm thinking we just update it to use `make docusaurus_start` from the main Makefile. Wdyt?